pkg/api/controllers/swagger.yaml

```diff
@@ -990,9 +975,6 @@ paths:
                     type: string
                     example: "invalid address"
 
-security:
```

Comment:
Why remove that?

Reply:
I tried to move it below `components` to see if this build problem could be fixed : `Non existing security scheme referenced: basicAuth. Skipping`. But unsuccessful, I put it back